packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/ProjectCard.js

```diff
@@ -130,11 +134,15 @@ export const ProjectCard = ({
         <Button
```

Comment:
This suggestion is optional - just if you are interested. 
It looks like all the logic is in the button and it would be nice if the ProjectCard was just a display component that wasn't aware of the access type etc. There are a couple of ways this would be made more streamlined but one method I think would work well is to use composed components which would invert some of the control to the ProjectCardList which already knows about the access types. You could export a `ProjectLockButton`, `ProjectPendingButton` and a `ProjectSelect` button and then use it in the ProjectCardList. 
eg.

```
import { ProjectCard, onSelectProject } from '../ProjectCard';

<ProjectCard Button={<ProjectButton action={onSelectProject} />} />